ui/app/manager/src/pages/page-configuration.ts

```diff
@@ -274,6 +327,44 @@ export class PageConfiguration extends Page<AppStateKeyed> {
                         </or-panel>
                         <or-panel .heading="${i18next.t("configuration.mapSettings").toUpperCase()}">
                             ${when(this.mapConfig, () => html`
+                                <div class="global-settings-container">
+                                    <div class="custom-server-group">
+                                        <div class="subheader">${i18next.t("configuration.global.tileServer")}</div>
+                                        <span>${i18next.t("configuration.global.tileServerDescription")}</span>
```

Comment:
We have a custom `<or-translate>` component that we use for translations like these.
It does a lot of things, but it also makes sure the text corrects itself when the user changes their language.
So you can replace all the `i18next.t("key")` appearances to `<or-translate value="key"></or-translate>`